liblepton/scheme/netlist/schematic-connection.scm

```diff
@@ -100,8 +101,12 @@ Example usage:
                  (_ #\?)))
              args)))))
 
+;;; This is a helper object property to get object connections
+;;; once.
+(define object-conns (make-object-property))
+
 (define (connected-to? object1 object2)
-  (not (not (memv object1 (object-connections object2)))))
+  (not (not (memv object1 (object-conns object2)))))
```

Comment:
Isn't double `not` superfluous here?

Reply:
Yes, in this particular case it is superfluous and may be omitted.  However, for user convenience, in case someone will debug the code in REPL, I wrote it so (for myself :-)). I believe, the guile compiler can deal with it nice and not profiled it to see how much it slows down the code ;-)